### PR TITLE
Format groups in report

### PR DESCRIPTION
**Pull request by Hwesta**
Display format groups in the formats table.  Add groups to the formats info in fixtures.  Should go in after #45

### CODE REVIEW

File: app/filters/aggregation.filter.js [hunk 1]
```diff
@@ -21,9 +21,11 @@
         if (!puid_data[record.puid]) {
           puid_data[record.puid] = {count: 0, size: 0};
         }
+        var format_info = Transfer.formats.filter(function (f) { return f.title == record.format; });
```
Comment (mistydemeo) [inline app/filters/aggregation.filter.js]: Nitpick, but `===` everywhere.

**Comment by mistydemeo** [comment]
Add a test to cover the new functionality?

File: app/filters/aggregation.filter.js [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
   angular.module('aggregationFilters', []).
 
-  filter('puid_data', function() {
+  filter('puid_data', function(Transfer) {
```
Comment (mistydemeo) [inline app/filters/aggregation.filter.js]: Use the array form of specifying deps:

``` js
filter('puid_data', ['Transfer', function(Transfer) {
// ...
}])
```

Comment (Hwesta) [inline app/filters/aggregation.filter.js]: Done

File: app/filters/aggregation.filter.js (hunk 1 above)
Comment (Hwesta) [inline app/filters/aggregation.filter.js]: Done

**Comment by Hwesta** [comment]
> Add a test to cover the new functionality?

Updated existing tests to check for format group.  Fixed a couple other test issues too.

**Comment by mistydemeo** [comment]
:+1:

**Pull request merged**